server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/SchemaSyncController.java

```diff
@@ -63,4 +67,16 @@ public ListResponse<TableComparing> skipTableComparing(@PathVariable Long id,
             @RequestBody SetSkippedReq setSkippedReq) {
         return Responses.list(shadowTableComparingService.setSkipTableComparing(id, setSkippedReq));
     }
+
+    @RequestMapping(value = "/structureComparison/{taskId}", method = RequestMethod.GET)
+    public SuccessResponse<DBStructureComparisonResult> listStructureComparisonResult(@PathVariable Long taskId,
```

Comment:
Why does this task require a separate controller instead of being placed in a flow related controller

Reply:
The existing `SchemaSyncController` is the controller of the interface for database synchronization related tasks. I think the structure comparison task is also part of the database synchronization task, so the interface for obtaining the structure comparison results is placed in `SchemaSyncController`, but the result of the structure comparison flowable task is still in the interface under `FlowInstanceController`